### PR TITLE
Updates templates working init project

### PR DESCRIPTION
**Pull request by elliotBraem**
- Fixes js template to reflect [quickstart.near](https://github.com/nearbuilders/quickstart)
- Removes ts template, it was unusable. We can save for a later ticket
- Removes test

### CODE REVIEW

File: lib/init.ts [hunk 1]
```diff
@@ -84,7 +84,7 @@ async function hasDirectory(dir) {
   }
 }
 
-function getDir(appName) {
-  const slug = slugify(appName, { lower: true });
+function getDir(ACCOUNT_ID) {
```
Comment (thisistheaj) [inline lib/init.ts:87]: if `ACCOUNT_ID` is a constant and we want to use that instead, that is fine. We should not have an argument to a function use this casing though.

File: tests/unit/init.ts [hunk 2]
```diff
@@ -1,7 +0,0 @@
-import { initProject } from "@/lib/init";
-
-describe('build', () => {
-  it('should build correctly', async () => {
-    await initProject('./__app_example_1', 'js-single');
```
Comment (thisistheaj) [inline tests/unit/init.ts:5]: If we do still want a test, I believe this should work (from my local changes):

```
describe('build', () => {
  it('should build correctly', async () => {
    await initProject('.', 'js');
  })
})
```

File: lib/init.ts (hunk 1 above)
Comment (thisistheaj) [reply lib/init.ts:87]: should either be changed to `accountId` or this function should take no parameters if the constant should always be used

File: lib/dev.ts [hunk 3]
```diff
@@ -260,7 +260,7 @@ async function generateDevJson(src: string, config: BaseConfig): Promise<DevJson
     }
 
     const widgetPath = path.relative(path.join(src, "src", "widget"), file).replace(ext, "");
-    const widgetKey = `${devAccount}/widget/${widgetPath.split("/").join(".")}`;
+    const widgetKey = `${devAccount}/widget/${widgetPath.split(path.sep).join(".")}`;
```
Comment (thisistheaj) [inline lib/dev.ts:263]: nice

**Review by thisistheaj — CHANGES_REQUESTED** [review]
thanks for putting this up — similar to the work I started. added some change requests where they differ. Feel free to override if I'm missing context on them.

File: tests/unit/init.ts (hunk 2 above)
Comment (elliotBraem) [reply tests/unit/init.ts:5]: describe('init' () => should init correctly

**Pull request merged**